components/clientComponents/forms/Review/Review.tsx

```diff
@@ -134,6 +186,16 @@ export const Review = ({ language }: { language: Language }): React.ReactElement
   // Focus heading on load
   useFocusIt({ elRef: groupsHeadingRef });
 
+  //Prior to useMemo, we grab the necessary translations for AddressComplete
+  addressCompleteStrings.unitNumber = t("addressComponents.unitNumber", { lng: language });
+  addressCompleteStrings.civicNumber = t("addressComponents.civicNumber", { lng: language });
+  addressCompleteStrings.streetName = t("addressComponents.streetName", { lng: language });
+  addressCompleteStrings.city = t("addressComponents.city", { lng: language });
+  addressCompleteStrings.province = t("addressComponents.province", { lng: language });
+  addressCompleteStrings.postalCode = t("addressComponents.postalCode", { lng: language });
+  addressCompleteStrings.country = t("addressComponents.country", { lng: language });
+  //This is done here, as useTranslation is inacessible inside useMemo.
```

Comment:
We should note this example as a case to think about when the team talks about patterns. The useMemo here (unrelated to this PR) was more of a nice to have and probably doesn't make much of a performance difference (would have to test more to be sure). So it could probably be removed to make the above easier. Also the react compiler will soon automatically search code for and insert useMemo's where it makes sense. Just a note, to note :)